translations/base-en.yaml

```diff
@@ -264,7 +264,7 @@ ingame:
         # The roman number for each tier
         tierLabels: [I, II, III, IV, V, VI, VII, VIII, IX, X]
 
-        maximumLevel: Maximum level
+        maximumLevel: MAXIMUM LEVEL (Speed x<currentMult>)
```

Comment:
Could you make this lowercase, e.g. `Maximum Level`? I prefer to have everything properly cased and then do `text-transform: uppercase`

Reply:
The reason I did this was because if I left text-transform: uppercase as is, I couldn't have "Speed x15" show up as lower case, as it would show as "SPEED X15", which I thought didn't look as nice. If you think it being all uppercase is fine, then I'm happy to change this to what you say, but just thought I would provide my reasoning here.

Reply:
Ah, makes sense. Hm I'm not too sure how much I like the formatting